src/backend/parser/parser.c

```diff
@@ -153,6 +153,12 @@ base_yylex(YYSTYPE *lvalp, YYLTYPE *llocp, core_yyscan_t yyscanner)
 		case PACKAGE:
 			cur_token_length = 7;
 			break;
+		case START:
+			cur_token_length = 5;
```

Comment:
Could you explain why should set the `cur_token_length` to 5 for `START` and 7 for `CONNECT`?